### DISCUSSION TITLE
进一步节省资源，在空闲时直接关闭模拟器

### DISCUSSION THREAD

**Discussion by DLmaster361** [LmeSzinc/StarRailCopilot#302] [Ideas]
目前已经支持在模拟器未开启时直接打开模拟器了，那么资源的优化为什么不更进一步，在完成任务后退出模拟器呢？

**Comment by mutr21** [comment]
可不可以直接像MAA一样定时启动,然后运行完自动关闭模拟器,而不是像ALAS一样自己填充时间.
因为崩铁我每天也就启动两次,而不是像碧蓝航线一样想要一直挂在电脑上.
因为好像有获取adb的功能能够重启模拟器和游戏,那么定时开启模拟器打开崩铁,完成之后关闭模拟器应该也好实现吧.
